src/components/Settings/Settings.js

```diff
@@ -1,9 +1,12 @@
 import { Link } from "@reach/router";
+import ButtonAppBar from "../AppBar";
 
 function Settings() {
   return (
     <>
-      <Link to="/home">Home</Link>
+      <Link to="/">Home</Link>
+
+      <ButtonAppBar title="Settings" />
```

Comment:
```suggestion
      <ButtonAppBar title="Einstellungen" />
```
Auch hier wieder Englisch